Resources/config/controller.xml

```diff
@@ -17,6 +17,7 @@
             <argument type="service" id="liip_monitor.helper.runner_manager" />
             <argument type="service" id="liip_monitor.helper" />
             <argument>%liip_monitor.view_template%</argument>
+            <argument type="service" id="logger" on-invalid="ignore" />
```

Comment:
I guess this needs to be mandatory .. we just need to ensure we have a logger to inject ..
I am not sure if the core logger service defaults to ``NullLogger`` if no logging is enabled .. if not we would need to create our own service which we optionally alias to the core logger via a compiler pass if the logger service is defined